File: Swift/Parallel/HtmlTextFormatter.swift
class HtmlTextFormatter {
  public func FormatAsBold(text: String) -> String {
    return "<b>(text)</b>"
  }
  public func FormatAsItalic(text: String) -> String {
    return "<i>\(text)</i>"
  }
  public func FormatAsSmall(text: String) -> String {
    return "<small>\(text)</small>"
  }
  
  public func FormatAsDeleted(text: String) -> String {
    return "<del>\(text)</del>"
  }
  public func FormatAsSubscript(text: String) -> String {
    return "<sub>\(text)</sub>"
  }
  public func FormatAsCustomTag(text: String, customTag: String) -> String {
    return "<\(customTag)>\(text)</\(customTag)>"
  }
}

File: Kotlin/Parallel/HtmlTextFormatter.kt
class HtmlTextFormatter {
public fun FormatAsBold(text: String): String = "<b>$text</b>"
public fun FormatAsItalic(text: String): String = "<i>$text</i>"
public fun FormatAsSmall(text: String): String = "<small>$text</small>"
public fun FormatAsDeleted(text: String): String = "<del>$text</del>"
public fun FormatAsSubscript(text: String): String = "<sub>$text</sub>"
public fun FormatAsCustomTag(text: String, customTag: String): String = "<$customTag>$text</$customTag>"
}
